### PR TITLE
[release-3.9] fix Service Catalog metrics issues

### PR DESCRIPTION
**Pull request by jboyd01**
Enable Service Catalog metric scrapes for Prometheus
includes fixes from https://github.com/openshift/origin/pull/18694  and https://github.com/openshift/origin/pull/18612

### CODE REVIEW

File: examples/prometheus/prometheus.yaml [hunk 1]
```diff
@@ -619,6 +619,21 @@ objects:
           action: keep
           regex: openshift-template-service-broker;apiserver;https
 
+      # Scrape config for the service catalog
+      - job_name: 'openshift-service-catalog'
+        scheme: http
```
Comment (jwforres) [inline examples/prometheus/prometheus.yaml]: we shouldn't be using insecure schemes for metrics endpoints

Comment (pmorie) [reply examples/prometheus/prometheus.yaml]: I agree, and didn't even think of this before. Where are we collecting information about go-live requirements like this?

**Comment by jboyd01** [comment]
I'm closing this in favor of https://github.com/openshift/origin/pull/19117 which is a revendor of upstream Service Catalog v0.1.11.  This contains changes to the Controller Manager that now listens over HTTPS and has appropriately signed SSL cert & key.

**Pull request closed without merging**